src/theme/default/colors.js

```diff
@@ -48,7 +48,7 @@ const appColors = {
   sectionHeaderBackground: rawColors.neutral.limedSpruce,
   sectionTitle: rawColors.neutral.bluebayoux,
   sectionDescription: rawColors.neutral.regentgrey,
-  placeholder: rawColors.neutral.iron,
+  placeholder: rawColors.neutral.regentgrey,
```

Comment:
we'll need to check if it looks ok everywhere it is used in cloud-fe, or did you already?

Reply:
Yes, indeed!
@christophidesp has already commited a changes needed for this update on the relevant PR I have opened on `cloud-frontend`! We know that we need to keep an eye if anything looks wired with this update and do the proper updates on `cloud-frontend`! 😄